src/App.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { BrowserRouter } from "react-router-dom";
```

Comment:
react-router-dom generate many bugs, I know it from experience and forums, e.g. 
content is not always loaded. Replace with react-router.